app/src/main/java/protect/card_locker/Group.java

```diff
@@ -5,36 +5,21 @@
 import androidx.annotation.Nullable;
 
 public class Group {
-    public final String _id;
+    public final int _id;
+    public final String name;
     public final int order;
 
-    public Group(final String _id, final int order) {
+    public Group(final int _id, final String name, final int order) {
         this._id = _id;
+        this.name = name;
         this.order = order;
     }
 
     public static Group toGroup(Cursor cursor) {
-        String _id = cursor.getString(cursor.getColumnIndexOrThrow(DBHelper.LoyaltyCardDbGroups.ID));
+        int _id = cursor.getInt(cursor.getColumnIndexOrThrow(DBHelper.LoyaltyCardDbGroups.ID));
+        String name = cursor.getString(cursor.getColumnIndexOrThrow(DBHelper.LoyaltyCardDbGroups.NAME));
         int order = cursor.getInt(cursor.getColumnIndexOrThrow(DBHelper.LoyaltyCardDbGroups.ORDER));
 
-        return new Group(_id, order);
-    }
-
-    @Override
-    public boolean equals(@Nullable Object obj) {
```

Comment:
Are we sure we lo longer need `.equals()` and `.hashCode()`? I think the default implementation is platform-dependent and would usually not consider objects equal if their fields are, only if they are the same object in memory. But I'm not familiar enough with the codebase to be sure that matters.